<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementsPreferences</class>
 <widget class="QDialog" name="MeasurementsPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="edit_name"/>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="labelSettlingTime">
     <property name="text">
      <string>Settling time</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxSettling"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="labelAverage">
     <property name="text">
      <string>Average Measures</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QSpinBox" name="spinBoxAverage">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="3">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="labelZero">
     <property name="text">
      <string>Zero</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QComboBox" name="combo_zero"/>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="labelMatrix">
     <property name="text">
      <string>Test type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QComboBox" name="combo_matrix">
     <property name="toolTip">
      <string>The type of test being made. Either in  midle section or bottom of the section</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="labelDVMTime">
     <property name="text">
      <string>Multimeter time</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QComboBox" name="combo_dvm_time">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="5">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Variable Control</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxAlphaStart"/>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxBetaStart"/>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMotorStart"/>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="radioButtonMotor">
        <property name="text">
         <string>Motor</string>
        </property>
        <attribute name="buttonGroup">
         <string>controlGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QRadioButton" name="radioButtonBeta">
        <property name="text">
         <string>Beta</string>
        </property>
        <attribute name="buttonGroup">
         <string>controlGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButtonAlpha">
        <property name="text">
         <string>Alpha</string>
        </property>
        <attribute name="buttonGroup">
         <string>controlGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxAlphaEnd">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxAlphaStep">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxBetaEnd">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMotorEnd">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxBetaStep">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMotorStep">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="labelStep">
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="labelEnd">
        <property name="text">
         <string>End</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelStart">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="radioButtonNone">
        <property name="text">
         <string>None</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string>controlGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="labelN">
        <property name="text">
         <string>Iterations</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSpinBox" name="spinBoxN">
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_name</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>doubleSpinBoxSettling</tabstop>
  <tabstop>spinBoxAverage</tabstop>
  <tabstop>combo_zero</tabstop>
  <tabstop>combo_matrix</tabstop>
  <tabstop>combo_dvm_time</tabstop>
  <tabstop>doubleSpinBoxAlphaStart</tabstop>
  <tabstop>doubleSpinBoxBetaStart</tabstop>
  <tabstop>doubleSpinBoxMotorStart</tabstop>
  <tabstop>doubleSpinBoxAlphaEnd</tabstop>
  <tabstop>doubleSpinBoxAlphaStep</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MeasurementsPreferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MeasurementsPreferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>test_input()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="controlGroup"/>
 </buttongroups>
</ui>
